### PR TITLE
upgrading celery==5.0.5

### PR DESCRIPTION
**Pull request by awais786**
upgrade celery==5.0.5

### CODE REVIEW

**Comment by awais786** [comment]
@pdpinch Please review.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 name: CI
-on: [push]
+on:
```
Comment (awais786) [inline .github/workflows/ci.yml:2]: Now tests will run against all prs.

**Review by umarmughal824 — APPROVED** [review]
👍

**Comment by umarmughal824** [comment]
@awais786 I have noticed that you have forked that repo and opened the PR from there. Normally, that code works fine and runs tests on our local repo PR's.

By specifying the branches and pull requests that way, I have doubt it would run the tests only against the master branch PRs.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (umarmughal824) [inline .github/workflows/ci.yml:2]:
```suggestion
on: [push, pull_request]
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+  push:
```
Comment (umarmughal824) [inline .github/workflows/ci.yml:3]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -1,5 +1,9 @@
...
+    branches: [ master ]
```
Comment (umarmughal824) [inline .github/workflows/ci.yml:4]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -1,5 +1,9 @@
...
+  pull_request:
```
Comment (umarmughal824) [inline .github/workflows/ci.yml:5]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -1,5 +1,9 @@
...
+    branches: [ master ]
```
Comment (umarmughal824) [inline .github/workflows/ci.yml:6]:
```suggestion
```

**Review by umarmughal824 — CHANGES_REQUESTED** [review]
Just a minor change.

**Comment by umarmughal824** [comment]
@awais786 we only need to update update on feature like `on: [push, pull_request]` to run the tests for the forked repository PR.

Here is the test PR for the test that the tests are running. https://github.com/mitodl/edx-sga/pull/294/files

**Review by umarmughal824 — APPROVED** [review]

**Pull request merged**